### PR TITLE
Add new docs articles

### PR DESCRIPTION
**Pull request by agsfer**
Entity Resolver Models and Features
Entity Resolver Model and NER Model Pairs
Relation Extraction Model and NER Model Pairs

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Code improvements with no or little impact
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [CONTRIBUTING](http://nlp.johnsnowlabs.com/contribute.html) page.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/en/entity_resolver_model_ner_model_pairs.md [hunk 1]
```diff
@@ -0,0 +1,190 @@
+---
+layout: docs
+header: true
+title: Relation Extraction Model and NER Model Pairs
```
Comment (muhammetsnts) [inline docs/en/entity_resolver_model_ner_model_pairs.md:4]: We need to change the title with `ENTITY RESOLVER MODEL AND NER MODEL PAIRS`.

File: docs/en/entity_resolver_model_ner_model_pairs_edited.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
+---
+layout: docs
+header: true
+title: Relation Extraction Model and NER Model Pairs
```
Comment (muhammetsnts) [inline docs/en/entity_resolver_model_ner_model_pairs_edited.md:4]: - We need to change the title with `ENTITY RESOLVER MODEL AND NER MODEL PAIRS`. 
- This file and `docs/en/entity_resolver_model_ner_model_pairs.md` seem the same.

File: docs/en/relation_extraction_model_pairs.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
+---
+layout: docs
+header: true
+title: Entity Resolver Models and Features
```
Comment (muhammetsnts) [inline docs/en/relation_extraction_model_pairs.md]: We need to change the title to `Relation Extraction Model and NER Model Pairs`.

**Review by muhammetsnts — CHANGES_REQUESTED** [review]

**Review by muhammetsnts — APPROVED** [review]
Thanks!

**Comment by vkocaman** [comment]
@muhammetsnts @agsfer lets out all these under one title so that we don't present all on the side bar..

title would be: `Best Practices in Using Pretrained Models in the Same Pipeline`

**Review by muhammetsnts — APPROVED** [review]

**Pull request merged**